CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
```

Comment:
```suggestion
### Fixed
```

Reply:
Oh, I wasn't sure there was fixed, https://keepachangelog.com/en/1.0.0/ Thanks for the input!

Reply:
Yes it can be useful especially for patch releases.